src/main/java/org/fundacionjala/coding/cesar/FizzBuzz.java

```diff
@@ -0,0 +1,32 @@
+package org.fundacionjala.coding.cesar;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ *
+ * @author admin-hp
+ */
+public class FizzBuzz {
+    private static final int FIVE = 5;
+    private static final int TREE = 3;
+    /**
+     * @param range List from int.
+     * @return list String replace.
+     */
+    public List<String> doBuzzFizz(final List<Integer> range) {
+
+        return range.stream().map(i -> i % FIVE == 0 & i % TREE == 0
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Correct this "&" to "&&". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2178)